app-crypt/efitools/ChangeLog

```diff
@@ -0,0 +1,35 @@
+# ChangeLog for app-crypt/efitools
```

Comment:
No need for this file in coreos-overlay, I like to keep the overlay pruned down to just the exact files and ebuilds required to make it clearer where to find our edits.